pkg/logger/ecs_error_log_test.go

```diff
@@ -0,0 +1,94 @@
+package logger
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zapcore"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+)
+
+var testErrorLogFilePath = "/opt/aws/aws-otel-collector/logs/aws-otel-collector-error-test.log"
+
+func TestWriteError(t *testing.T) {
+	defer os.Remove(testErrorLogFilePath)
+	openExistingOrNewErrorFile = OpenExistOrNew
+	entry := zapcore.Entry{
+		Message: "test",
+	}
+	_, error := WriteError(entry, testErrorLogFilePath)
```

Comment:
Why this test case only cover one output of `WirteError` function?

Reply:
oh yeah, the test part still not all done yet, will add some more test case later